packages/components/src/presentation-components/properties/inputs/AsyncSelect.tsx

```diff
@@ -129,6 +129,7 @@ export function AsyncSelect<OptionType, Group extends GroupBase<OptionType>, Add
           indicatorSeparator: (base) => ({ ...base, marginTop: undefined, marginBottom: undefined, margin: "0 var(--iui-size-xs)" }),
           clearIndicator: () => ({}),
           dropdownIndicator: () => ({}),
+          placeholder: () => ({ width: "100", display: "block", whiteSpace: "nowrap", textOverflow: "ellipsis", overflow: "hidden" }),
```

Comment:
Shouldn't it be `100%`?
```suggestion
          placeholder: () => ({ width: "100%", display: "block", whiteSpace: "nowrap", textOverflow: "ellipsis", overflow: "hidden" }),
```